### PR TITLE
Add 'mirrored-' to mirrored charts for 2.5.6

### PR DESCRIPTION
**Pull request by thedadams**
Issue:
https://github.com/rancher/rancher/issues/30888

### CODE REVIEW

File: packages/rancher-tracing/package.yaml
Comment (brendarearden) [inline packages/rancher-tracing/package.yaml:2]: is this one supposed to be RC version increase or did we do package version for a reason here?

Comment (thedadams) [reply packages/rancher-tracing/package.yaml:2]: I tried to bump the RC version on this one and it failed validation `new-assets/charts-without-rc/rancher-tracing/rancher-tracing/1.20.001 does not match that of old contents at original-assets/charts/rancher-tracing/rancher-tracing/1.20.001`. I bumped the package version since the error message references "without-rc".

Comment (cbron) [reply packages/rancher-tracing/package.yaml:2]: 1.20.001 was already released: https://github.com/rancher/charts/tree/main/assets/rancher-tracing
So this is working as intended.
@aiyengar2 you had listed this as new version, is there problem there ? Hoping it got generated correctly.

**Comment by cbron** [comment]
Adding maintainers to cover your areas

File: packages/rancher-logging/package.yaml
Comment (nickgerace) [inline packages/rancher-logging/package.yaml:3]: This will need to be `08` based on https://github.com/rancher/charts/pull/995#pullrequestreview-593715679
I believe that PR will be merged first since it has committer approval.

**Review by nickgerace — CHANGES_REQUESTED** [review]

File: packages/rancher-tracing/package.yaml
Comment (brendarearden) [reply packages/rancher-tracing/package.yaml:2]: gotcha okay then that should be fine!

**Review by brendarearden — DISMISSED** [review]
Istio/Kiali/Jaeger changes all look fine

File: packages/rancher-logging/package.yaml
Comment (thedadams) [reply packages/rancher-logging/package.yaml]: Updated.

**Review by cbron — COMMENTED** [review]
Lgtm, just need to figure out tracing situation and merge mirror pr

**Review by nickgerace — DISMISSED** [review]
LGTM. A bit strange that logging's previous RC is still 06, not 07 though

**Comment by cbron** [comment]
Its cause this hasn't been rebased lately. There are conflicts.

File: packages/rancher-tracing/package.yaml
Comment (cbron) [reply packages/rancher-tracing/package.yaml:2]: Sounds like there were no changes. So we will just bump this to 1.20.002, I added that to the "new charts" issue.

**Review by cbron — APPROVED** [review]

**Review by nickgerace — APPROVED** [review]
LGTM with the recent rebase

File: packages/rancher-tracing/package.yaml
Comment (aiyengar2) [reply packages/rancher-tracing/package.yaml:2]: Yes there was no need for a change for rancher-tracing since it was a local chart with no CRD chart annotation (so the only change was to the rcVersion and make sync did not fail in the initial migration of the charts).

LGTM

**Review by aiyengar2 — APPROVED** [review]
Monitoring LGTM

**Pull request merged**
